egs/aishell/asr/simple_v1/prepare.py

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python3
 
-# Copyright (c)  2020  Xiaomi Corporation (authors: Junbo Zhang, Haowen Qiu)
+# Copyright (c)  2020  Xiaomi Corporation (authors: Junbo Zhang, Haowen Qiu, Pingfeng Luo)
```

Comment:
Are you also from Xiaomi? Both of the authors are from Xiaomi. That's why their
names are listed together.